app/src/main/java/org/jellyfin/androidtv/data/service/BackgroundService.kt

```diff
@@ -181,10 +182,17 @@ class BackgroundService(
 						.centerCrop()
 						.submit()
 				}
-				.map { future -> async { future.get() } }
+				.map { future -> async {
+					try {
+						future.get()
+					} catch (ex: GlideException) {
+						Timber.e(ex, "There was an error fetching the background image:")
+						null
+					}
+				} }
```

Comment:
Running the built-in formatter changes this code slightly:
```suggestion
				.map { future ->
					async {
						try {
							future.get()
						} catch (ex: GlideException) {
							Timber.e(ex, "There was an error fetching the background image:")
							null
						}
					}
				}
```

I also think the message shouldn't contain a semicolon. The message should tell what issue happened, the exception is additional information to that.

Reply:
I fixed it and pushed it in this commit; https://github.com/jellyfin/jellyfin-androidtv/pull/774/commits/bb734f00a3d5b18dbfc8dae28773e5dc59b72376